src/AuthLDAP.php

```diff
@@ -1926,6 +1926,7 @@ public static function searchForUsers(
                             ),
                             E_USER_WARNING
                         );
+                        throw new \RuntimeException(ldap_error($ds));
```

Comment:
So you should jst replace the `trigger_error` call with throw `new \RuntimException`; the `buildError` method adds all required extra information needed. 
Also, I had in mind a trigger_error was fatal, thus new exception could not be reached...

Reply:
`trigger_error` is not a blocker as long as the level is not `E_USER_ERROR`.